src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.hpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//==== ONNXToTosaLegalizeUtils.hpp - ONNX dialects to TOSA lowering Utils-===//
+//
+// Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+// Copyright (c) 2022 Advanced Micro Devices, Inc.
+//
+// =============================================================================
+//
+// This file contains common utils shared by the functions performing the
+// lowering to the TOSA dialect. It is also used by TensorFlow and Pytorch.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ONNXMLIR_CONVERSION_ONNXTOTOSA_TOSALEGALIZEUTILS_H
+#define ONNXMLIR_CONVERSION_ONNXTOTOSA_TOSALEGALIZEUTILS_H
+
+#include "mlir/Dialect/Quant/QuantTypes.h" // from @llvm-project
+#include "mlir/Dialect/Tosa/IR/TosaOps.h"
```

Comment:
Is the `// from @llvm-project` annotation useful? And if so, why isn't it on `"mlir/Dialect/Tosa/IR/TosaOps.h"`?

Reply:
Good question, I think this is a relic from adopting code from TF. I will delete the comments